scripts/add-new-version.js

```diff
@@ -99,6 +99,7 @@ async function addNewVersion(newVersion) {
   await execute(`cp -R ./examples/${latestExampleVersion} ${newExampleVersionDir}`);
   
   // Replace $ref and $id paths such as `/3.0.0/` with new version (http://asyncapi.com/definitions/3.0.0/specificationExtension.json)
+  /* eslint-disable no-useless-escape */
   await execute(`find ${newVersionDir} -name '*.json' -exec sed -i '' \"s+\/${latestVersion}\/+\/${newVersion}\/+g\" {} +`);
```

Comment:
Let's fix the issue instead of hiding it:
```suggestion
  /* eslint-disable no-useless-escape */
  await execute(`find ${newVersionDir} -name '*.json' -exec sed -i '' "s+/${latestVersion}/+/${newVersion}/+g" {} +`);
```